typing/includemod.ml

```diff
@@ -1412,3 +1416,32 @@ let check_modtype_equiv ~loc env id mty1 mty2 =
       raise (Error(env,
                    Error.(In_Module_type_substitution (id,diff mty1 mty2 e)))
             )
+
+module Check = struct
+  type 'a compatibility_test = Env.t -> Subst.t -> 'a -> 'a -> bool
+  open Core_inclusion
+
+  let check_only f env subst l r =
+    let loc = Location.none and id = Ident.create_local "*dummy*" in
```

Comment:
I wonder why the function wants a location, but we provide a dummy one. Is the intention that we may pass a better location from outside later on, or maybe that the location does not matter for that particular usage of the function?